### ISSUE TITLE
Last time slice of output history file has time index of -Infinityf

### ISSUE THREAD

**Issue by orianac** [UW-Hydro/RVIC#63]
When running RVIC across multiple sites, the output history file has a time index that ends with this (taken from output of `ncdump -c -t` on the file):
```
    ...734490.5, 734491.5, 734492.5, 734493.5, 734494.5, 734495.5, 734496.5,
    734497.5, 734498.5, 734499.5, 734500.5, 734501.5, 734502.5, 734503.5,
    -Infinityf ;
```

The last few timestamps of the forcing file look like this:
```
    "2011-12-17", "2011-12-18", "2011-12-19", "2011-12-20", "2011-12-21",
    "2011-12-22", "2011-12-23", "2011-12-24", "2011-12-25", "2011-12-26",
    "2011-12-27", "2011-12-28", "2011-12-29", "2011-12-30", "2011-12-31"
```

Relevant parts of the convolution config file:
```
[HISTORY]
#--Number of History tapes.
RVICHIST_NTAPES : 1
#--Per tape series maximum number of time samples.
RVICHIST_MFILT  : 100000
#--Per tape series history file density (i.e. output precision)
#    1=double precision
#    2=single precision
RVICHIST_NDENS  : 1
#--Per tape series history write frequency.
#  positive means in time steps
#  0 = monthly
#  negative means hours
RVICHIST_NHTFRQ : 1
#--Per tape series history write frequency.
#  A - Average, over the output interval.
#  I - Instantaneous, output the value at the output interval.
#  X - Maximum, over the output interval.
#  M - Minimum, over the output interval.
RVICHIST_AVGFLAG : A
#-- Per tape series history file format (char)
# Valid Values: grid, array
RVICHIST_OUTTYPE: array
RVICHIST_NCFORM: NETCDF3_CLASSIC
#-- Per tape series output units (char)
# Valid Values: kg m-2 s-1, m3/s
RVICHIST_UNITS: m3/s
```

**Comment by bartnijssen** [comment]
Are there data values associated with the `-Infinity` timestamp? I.e. is this one extra (and should just be removed) or is it needed and just doesn't have a value?

**Comment by jhamman** [comment]
My bet is that this is an extra value:
```python
In [1]: from netCDF4 import num2date

In [2]: num2date(734503.5, "days since 0001-01-01")
Out[2]: datetime.datetime(2011, 12, 31, 12, 0)
```

I have not tested many of the combinations of options for the `RVICHIST_MFILT` (e.g. `RVICHIST_MFILT  : 100000`).

My best guess is that we're missing a `-1` somewhere in determining the size of the time array.

**Comment by orianac** [comment]
The timestamp is extra (I don't need it and am currently, as a workaround, removing it after having read it into python). It has junk zeros for data for that timestamp.

**Comment by jhamman** [comment]
closed via #71. 

cc @orianac
